src/linker/Linker/Message.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mono.Linker
+{
+	public class Message
```

Comment:
This is not "Message" but container of more things. The name should reflect that.